sample-code/examples/ruby/cucumber_ios_web/features/step_definitions/steps.rb

```diff
@@ -0,0 +1,37 @@
+# These are the 'step definitions' which Cucumber uses to implement features.
+#
+# Each step starts with a regular expression matching the step you write in
+# your feature description.  Any variables are parsed out and passed to the
+# step block.
+#
+# The instructions in the step are then executed with those variables.
+#
+# The '$driver' object is the appium_lib driver, set up in the cucumber/support/env.rb
+# file, which is a convenient place to put it as we're likely to use it often.
+# For more on step definitions, check out the documentation at
+# https://github.com/cucumber/cucumber/wiki/Step-Definitions
+#
+# For more on rspec assertions, check out
+# https://www.relishapp.com/rspec/rspec-expectations/docs
+Given(/^I have entered ([^"]*) into Email field$/) do |value|
+  @driver.find_element(id:'fbemail').send_keys(value)
+  wait(timeout:2,message:'Text not entered into email').until { @driver.find_element(id:'fbemail').attribute('value').eql?value }
+end
+
+And(/^I have entered ([^"]*) into Comments field$/) do |value|
+  @driver.find_element(id:'comments').send_keys(value)
+  wait(timeout:2,message:'Text not entered into comments').until { @driver.find_element(id:'comments').attribute('value').eql?value }
```

Comment:
selenium has a wait, the appium ruby bindings have a wait, let's not create a 3rd type of wait.